pkg/kubelet/volumes.go

```diff
@@ -229,6 +231,15 @@ func (kl *Kubelet) getPodVolumesFromDisk() map[string]cleanerTuple {
 		volumes, err := kl.getPodVolumes(podUID)
 		if err != nil {
 			glog.Errorf("%v", err)
```

Comment:
@ncdc  - I'm thinking we just remove it since it's just duplicating the exact same log message that we already logged in getPodVolumes - agree?